trio/_tests/test_channel.py

```diff
@@ -45,26 +49,26 @@ async def test_channel():
     with pytest.raises(trio.ClosedResourceError):
         await r.receive()
     with pytest.raises(trio.ClosedResourceError):
-        await r.receive_nowait()
+        r.receive_nowait()
     await r.aclose()
```

Comment:
why the change of functionality?

Reply:
Because mypy said that `receive_nowait` is not an awaitable function and raises an error if attempted